browser/src/layer/vector/SVGGroup.js

```diff
@@ -258,7 +258,7 @@ L.SVGGroup = L.Layer.extend({
 	_hasVisibleEmbeddedSVG: function () {
 		var result = false;
 		this._forEachSVGNode(function (svgNode) {
-			if (svgNode.getAttribute('opacity') !== 0)
+			if (svgNode.getAttribute('opacity') != 0)
```

Comment:
hmm, maybe better idea would be to use strict inequality after cast - parseInt()  ?

Reply:
`try ... catch` for parsing error inside loop will be then needed probably.

Reply:
parseInt does not throw exceptions (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt). On erroneous cases it returns NaN which would fail strict comparison checks.